### PR TITLE
Support for UI of MultimodalRAGWithVideos in GenAIExamples

### PR DESCRIPTION
**Pull request by tileintel**
## Description

This PR updates LVM and multimodal retriever to support for UI of MultimodalRAGwithVideos in GenAIExamples. 
This is required for v1.0 release.

## Issues

N/A

## Type of change

List the type of change like below. Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would break existing design and interface)
- [x] Others (enhancement, documentation, validation, etc.)

## Dependencies

N/A

## Tests

N/A

### CODE REVIEW

**Comment by tileintel** [comment]
@lvliang-intel @XinyuYe-Intel @Spycsh Can you please help to review and merge this PR? It turns out that we need more metadata for UI as proofs for generated answers, so I have implemented this PR. This is required for v1.0 release. We need this in order to test GenAIExample PR. 
Thanks a lot.

File: comps/lvms/tgi-llava/lvm_tgi.py [hunk 1]
```diff
@@ -46,6 +48,11 @@ async def lvm(request: Union[LVMDoc, LVMSearchedMultimodalDoc]) -> TextDoc:
         if logflag:
             logger.info("[LVMSearchedMultimodalDoc ] input from retriever microservice")
         retrieved_metadatas = request.metadata
+        if retrieved_metadatas is None or len(retrieved_metadatas) == 0:
```
Comment (Spycsh) [inline comps/lvms/tgi-llava/lvm_tgi.py:51]: `if not retrieved_metadatas:` should be more concise

**Review by Spycsh — APPROVED** [review]

**Comment by Spycsh** [comment]
Please fix potential CI issues.

**Comment by tileintel** [comment]
> Please fix potential CI issues.
@Spycsh 
We pretty need this PR merged soon so that we can submit PR for GenAIExamples. Please instruct how to handle the failed tests. A lot of tests failed are not due to this PR (to the best of my knowledge). Also, I don't know why this PR triggered all non-related tests (not happened in the previous other PRs). For example, one of the failed tests due to 
  `File "/home/user/.local/lib/python3.11/site-packages/pinecone/config/config.py", line 58, in build
    raise PineconeConfigurationError("You haven't specified an Api-Key.")
pinecone.exceptions.exceptions.PineconeConfigurationError: You haven't specified an Api-Key.` 
I have no idea why this relates to this PR. 
I believe a lot of tests are not rooted from this PR but other previously merged PR. Please instruct how we identify our related failed tests?

File: comps/lvms/tgi-llava/lvm_tgi.py (hunk 1 above)
Comment (tileintel) [reply comps/lvms/tgi-llava/lvm_tgi.py]: Thank for your feedback. @Spycsh 
I have updated it to be `if not retrieved_metadatas:`

**Comment by Spycsh** [comment]
> > Please fix potential CI issues.
> > @Spycsh
> > We pretty need this PR merged soon so that we can submit PR for GenAIExamples. Please instruct how to handle the failed tests. A lot of tests failed are not due to this PR (to the best of my knowledge). Also, I don't know why this PR triggered all non-related tests (not happened in the previous other PRs). For example, one of the failed tests due to
> > `File "/home/user/.local/lib/python3.11/site-packages/pinecone/config/config.py", line 58, in build     raise PineconeConfigurationError("You haven't specified an Api-Key.") pinecone.exceptions.exceptions.PineconeConfigurationError: You haven't specified an Api-Key.`
> > I have no idea why this relates to this PR.
> > I believe a lot of tests are not rooted from this PR but other previously merged PR. Please instruct how we identify our related failed tests?

cc. @chensuyue

**Comment by chensuyue** [comment]
> > > Please fix potential CI issues.
> > > @Spycsh
> > > We pretty need this PR merged soon so that we can submit PR for GenAIExamples. Please instruct how to handle the failed tests. A lot of tests failed are not due to this PR (to the best of my knowledge). Also, I don't know why this PR triggered all non-related tests (not happened in the previous other PRs). For example, one of the failed tests due to
> > > `File "/home/user/.local/lib/python3.11/site-packages/pinecone/config/config.py", line 58, in build     raise PineconeConfigurationError("You haven't specified an Api-Key.") pinecone.exceptions.exceptions.PineconeConfigurationError: You haven't specified an Api-Key.`
> > > I have no idea why this relates to this PR.
> > > I believe a lot of tests are not rooted from this PR but other previously merged PR. Please instruct how we identify our related failed tests?
> 
> cc. @chensuyue

Oh, because this PR changed `__init__.py`, which has been recognized as a common used scripts, so all the test will be trigger. And yes, that seems not necessary, I will submit a PR to omit the test.

**Comment by chensuyue** [comment]
The test scope has been optimized.

**Comment by tileintel** [comment]
@lvliang-intel Can you please review this PR for v1.0 release? Thanks

**Review by lvliang-intel — APPROVED** [review]

**Pull request merged**